playpen/ansible/roles/dev/files/bashrc

```diff
@@ -35,9 +35,32 @@ pstatus() {
 
 preset() {
     pstop;
+
+    pushd ~/devel/
+    # Remove and unlink all development files
+    for r in {pulp_deb,pulp_docker,pulp_openstack,pulp_ostree,pulp_puppet,pulp_python,pulp_rpm,pulp}; do
```

Comment:
Suggestion: since this PR is adding two more instances of this repeated list, I think now is a good time to make this list a variable. Otherwise when we add new plugins, we have to hunt down this list in many places. I know this list is also repeated in other places (Ansible and dev-setup.sh for example) and that it would be nice to solve this problem once and for all globally. However, I think for now that just making it a variable in only this file and only solving the problem for this file would be an improvement. We can think about the bigger problem later.

I also suggest leaving "pulp" out of the variable, since you need to uninstall pulp last and install it first and it would be nice to not have two variables to represent those two orderings.